vue-frontend/src/components/about/HowItAllStartedSection.vue

```diff
@@ -1,19 +1,23 @@
 <template>
-  <div class="xll:tw-container !tw-mt-20 xl:tw-mb-20 2xl:tw-mb-48">
+  <div class="!tw-mt-20 xl:tw-mb-20 2xl:tw-mb-48">
     <h1
       class="tw-text-center tw-text-[1.875rem] md:tw-text-[2.813rem] tw-leading-[2.25rem] md:tw-leadig-[3.875rem] tw-font-inter-bold tw-text-black-core/[0.87]"
     >
       How it all started?
     </h1>
-    <div class="tw-flex tw-flex-row xl:tw-pt-20">
-      <div class="tw-w-[60%] tw-flex tw-flex-col tw-overflow-hidden tw-py-40">
+    <div
+      class="tw-flex tw-flex-row xl:tw-pt-20 tw-w-full tw-items-center tw-mx-auto 2xl:tw-w-[1535px] xll:tw-w-[1530px] 3xl:tw-w-[1595px]"
```

Comment:
Let's follow the ordering of classes
Also make sure MR is following review checklists